ci-operator/step-registry/openshift-qe/orion/olmv1/openshift-qe-orion-olmv1-commands.sh

```
@@ -198,34 +198,40 @@ orion_exit_status=$?
 set -e
 
 if [[ "$OUTPUT_FORMAT" == "JUNIT" ]]; then
+  # Orion appends _{test_name} to --save-output-path, so junit.xml becomes junit_<name>.xml
+  JUNIT_FILE=$(ls junit_*.xml 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)